lua/nvim_lsp/util.lua

```diff
@@ -7,6 +7,8 @@ local fn = vim.fn
 
 local M = {}
 
+M.Severity = vim.tbl_extend("keep", lsp.protocol.MessageType, { None = 0 })
```

Comment:
not in favor of this. Adding new types that have nearly the same semantics as other types is a cognitive burden.